library/lib_window.ks

```diff
@@ -0,0 +1,26 @@
+
+
+@lazyglobal off.
+
+function create_window{
```

Comment:
What interface you are using? github.com hides comments assigned to "outdated diff".